content/proposals.md

```diff
@@ -185,6 +185,15 @@ is as follows:
     -   Take care in creating your proposal. Specify your intended
         changes, and give reasoning to back them up. Changes without
         justification will likely be poorly received by the community.
+    -   Proposals must include a migration plan for how to switch
+        from unstable, prefixed implementations to stable, unprefixed ones
+        in both client and server applications.
+        Such plan should include specifics like:
+        - How to handle immutable data (e.g. past unstable events) that can't
+          be migrated.
+        - How to switch from unstable to stable push rules.
+        - When (not a precise point in time, but rather after what event) clients
+        are allowed to stop sending unstable events.
```

Comment:
not all MSCs even use unstable identifiers, so I think this needs some more qualification.

It also feels odd to include this requirement before introducing the concept of unstable identifiers, which doesn't happen until way down [here](https://github.com/matrix-org/matrix-spec/blob/e1aee24bbfe051512dd19c1031489703aa8135fd/content/proposals.md#early-release-of-an-mscidea)

Reply:
Indeed, this can be made better, though perhaps [this](https://github.com/matrix-org/matrix-spec/pull/1650#discussion_r1340237997) should be addressed first.